LICENSE-binary

```diff
@@ -321,6 +321,9 @@ org.apache.kerby:kerby-pkix:1.0.1
 org.apache.kerby:kerby-util:1.0.1
 org.apache.kerby:kerby-xdr:1.0.1
 org.apache.kerby:token-provider:1.0.1
+org.apache.solr:solr-solrj:8.8.2
+org.apache.solr:solr-core:8.8.2
+org.apache.solr:solr-test-framework:8.8.2
```

Comment:
Ah, I guess I missed this while bumping solr version (while fixing maven 3.8 issues).